app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%=
+    per_page_options = [5, 10, 15, 25, 50]
+    per_page_options << Setting[:entries_per_page].to_i
+    per_page_options << params[:per_page].to_i unless params[:per_page].blank?
```

Comment:
note that currently many (most?) ui controllers ignore the `per_page` parameter.